internal/clients/team2/actions.go

```diff
@@ -203,3 +203,10 @@ func (a *AgentTwo) Ping() {
 	// We treat this as a social event and update the Network parameter in Social Capital.
 	a.UpdateSocNetBike(1.0, SocialEventWeight_AgentOnBike)
 }
+
+func (a *AgentTwo) ToggleOnBike() {
+	// If we are about to toggle off a bike, send a KickOff message.
+	a.SendKickOffMessage(a.onBike, a.GetID())
```

Comment:
Idk if we need to have a kickoff message if we're changing bike willingly rather than being kicked out.